setup.py

```diff
@@ -49,7 +49,8 @@ def get_test_suite():
             'acl-loader = acl_loader.main:cli',
             'sonic-clear = clear.main:cli',
             'debug = debug.main:cli',
-            'undebug = undebug.main:cli'
+            'undebug = undebug.main:cli',
```

Comment:
While you're at it, could you please rearrange the `console-scripts` in alphabetical order, also? Thanks.

Reply:
Done